src/secex-genprotimgvm-scripts/genprotimg.bu

```diff
@@ -74,14 +74,16 @@ storage:
           ln -sf "../genprotimg.service" "${UNIT_DIR}/default.target.wants/"
 systemd:
   units:
-    - name: serial-getty@.service
-      dropins:
-        - name: autologin-core.conf
-          contents: |
-            [Service]
-            TTYVTDisallocate=no
-            ExecStart=
-            ExecStart=-/usr/sbin/agetty --autologin core  -o '-p -f core' --noclear %I $TERM
-      mask: false
-    - name: autovt@.service
-      mask: false
```

Comment:
Why did we drop the autologin? Can we put that in a separate commit with reasoning?

Reply:
I'd like to provide this config to guys from IBM and switch official VM  from `ubuntu` (as i remember) to `rhcos`. And official VM does nothing but `genprotimg` + `poweroff`.